README.md

```diff
@@ -191,6 +192,16 @@ $faker->validKppCode(); // 773301001 7733AZ001
 $faker->invalidKppCode(); // 7733010011 77330100Z
 ```
 
+### `AvtoDev\FakerProviders\Providers\Identifiers\CadastralNumberProvider`
+
+```php
+<?php /** @var \Faker\Generator|\AvtoDev\FakerProviders\ExtendedFaker $faker */
+
+$faker->cadastralNumber(); // 66:41:153222:68
+$faker->validCadastralNumber(); // 77:22:5874698:1
```

Comment:
Are you sure about this value is really valid?

Reply:
It is valid, but it may not exist